### PR TITLE
Change ChromeDriver package from buggy one to better maintained one

### PR DESCRIPTION
**Pull request by NickGrantONS**
# Motivation and Context
The PyPi package https://pypi.org/project/chromedriver_installer hasn't been maintained for 2 years, and the single contributor has failed to approve a pull request, which would fix a bug, due to the code screen-scraping a webpage to find the latest version of the ChromeDriver, which Google has changed.

# What has changed
Replace `chromedriver_installer` with `chromedriver-binary`. Add import to the init script so that the path is correctly set up for Selenium.

# How to test?
Run the acceptance tests. You should notice no difference.

# Links
Trello: https://trello.com/c/wyNCj7iF/738-bug-fix-the-ci-cd-pipeline-and-acceptance-tests
Old package fix pull request (unapproved): https://github.com/authomatic/chromedriver_installer/pull/10
New package: https://pypi.org/project/chromedriver-binary/

### CODE REVIEW

File: acceptance_tests/features/pages/sign_in_internal.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
+import time
+
 from acceptance_tests import browser
 from config import Config
 
 
 def go_to():
     browser.visit(Config.RESPONSE_OPERATIONS_UI + '/sign-in')
+    time.sleep(2)
```
Comment (gavinleeedwards) [inline acceptance_tests/features/pages/sign_in_internal.py]: What does this have to do with the PR?

Comment (NickGrantONS) [reply acceptance_tests/features/pages/sign_in_internal.py]: For some reason, the acceptance tests were failing without this. Take it out and try it yourself.
